WalletWasabi.Fluent/ViewModels/SearchBar/Sources/ActionsSource.cs

```diff
@@ -0,0 +1,59 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Reactive.Linq;
+using System.Threading.Tasks;
+using DynamicData;
+using WalletWasabi.Fluent.ViewModels.Navigation;
+using WalletWasabi.Fluent.ViewModels.SearchBar.Patterns;
+using WalletWasabi.Fluent.ViewModels.SearchBar.SearchItems;
+
+namespace WalletWasabi.Fluent.ViewModels.SearchBar.Sources;
+
+public class ActionsSource : ISearchItemSource
+{
+	public IObservable<IChangeSet<ISearchItem, ComposedKey>> Changes => GetItemsFromMetadata()
+		.ToObservable()
+		.ToObservableChangeSet(x => x.Key);
+
+	private static IEnumerable<ISearchItem> GetItemsFromMetadata()
+	{
+		return NavigationManager.MetaData
+			.Where(m => m.Searchable)
+			.Select(m =>
+			{
+				var func = CreateFunc(m);
+				var searchItem = new ActionableItem(m.Title, m.Caption, func, m.Category ?? "No category", m.Keywords)
+				{
+					Icon = m.IconName,
+					IsDefault = true,
+				};
+				return searchItem;
+			});
+	}
+
+	private static Func<Task> CreateFunc(NavigationMetaData navigationMetaData)
```

Comment:
Something smells to me. What happened with this fix? https://github.com/zkSNACKs/WalletWasabi/pull/7785/files
Is it somewhere else?

Reply:
Locations changed and file was renamed, the code was lost :( 
Now it's fixed. Thanks for noticing!